cypress/e2e/undo-redo/block-configs-should-update-flow-state/branching-block-configs.cy.js

```diff
@@ -0,0 +1,48 @@
+describe('Mutate flow state for the case (branching) block', () => {
+  const EXIT_NAME = 'My Exit'
+  const EXIT_EXPRESSION = 'TRUE'
+
+  const DEFAULT_EXIT_NAME = 'Default'
+  const DEFAULT_EXIT_EXPRESSION = ''
+
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'case block test',
+    })
+
+    cy.addBlock(['Branching']).as('branchingBlock')
+
+    cy.get('[data-cy="add-exit--btn"]').click()
+    cy.get('[data-cy="advanced-exit-name--input"]').type(EXIT_NAME)
+    cy.get('[data-cy="advanced-exit-test-expression--input"]').type(EXIT_EXPRESSION)
+  })
+
+  it('should add choices', () => {
```

Comment:
would it be better to call this 'exits' rather than 'choices'?
(and the same for another `it`?)